analytics/src/main/java/com/segment/analytics/SegmentIntegration.java

```diff
@@ -98,6 +105,8 @@ public String key() {
   private final Cartographer cartographer;
   private final ExecutorService networkExecutor;
   private final ScheduledExecutorService flushScheduler;
+
+  private final HashMap<String, AttemptGoalPayload> attemptGoalPayloads;
```

Comment:
Add a comment explaining what this map is for.

Reply:
done